core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/ServicePerimeterApiService.scala

```diff
@@ -26,7 +26,7 @@ trait ServicePerimeterApiService extends UserInfoDirectives {
     path("servicePerimeters" / Segment / "projects" / Segment) { (servicePerimeterName, projectId) =>
       put {
         complete {
-          userServiceConstructor(userInfo).addProjectToServicePerimeter(ServicePerimeterName(URLDecoder.decode(servicePerimeterName, UTF_8.name)), RawlsBillingProjectName(projectId))
+          userServiceConstructor(userInfo).addProjectToServicePerimeter(ServicePerimeterName(URLDecoder.decode(servicePerimeterName, UTF_8.name)), RawlsBillingProjectName(projectId)) // todo: the `URLDecoder.decode` on this line decodes url-encoded chars. is this ok or could it lead to security issues, such as stored/persistent XSS?
```

Comment:
this isn't a PPW change. I don't recall the exact places where we are encoding and decoding, but I'm pretty sure we've been doing this since service perimeters were introduced. Not saying that it isn't important to get right, but I want to make sure we're staying focused on bugs that may be introduced by PPW